api/index.md

```diff
@@ -4,39 +4,30 @@ title: API
 nav_order: 20
 has_children: true
 ---
+
 # Overview
 
-The [Data Commons Graph](https://datacommons.org) aggregates data from many
+The [Data Commons graph](https://datacommons.org) aggregates data from many
```

Comment:
I think should put the graph out of this link, since the link is about "Data Commons", but not the graph?

Reply:
Agreed.